packages/transaction-controller/src/TransactionController.ts

```diff
@@ -1695,6 +1730,164 @@ export class TransactionController extends BaseController<
       currentNetworkIsEIP1559Compatible && currentAccountIsEIP1559Compatible
     );
   }
+
+  private async updatePostTransactionBalance(
```

Comment:
Since this and the other new private methods are a reasonable amount of logic, and they only really update the transaction metadata, is it worth moving them to `utils/swaps.ts`?

We could move all the new constants there also as there is a lot but very swaps specific.

Reply:
This is moved in https://github.com/MetaMask/core/pull/1877/commits/127444c3b779230c85656050ea48613c3ff3de71